sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsModelFactory.cs

```diff
@@ -219,7 +219,7 @@ public static AnalyzeSentimentResultCollection AnalyzeSentimentResultCollection(
         public static DetectedLanguage DetectedLanguage(string name, string iso6391Name, double confidenceScore, IList<TextAnalyticsWarning> warnings = default)
         {
             warnings ??= new List<TextAnalyticsWarning>();
-            return new DetectedLanguage(new DetectedLanguageInternal(name, iso6391Name, confidenceScore), warnings);
+            return new DetectedLanguage(new DetectedLanguageInternal(name, iso6391Name, confidenceScore, default), warnings);
```

Comment:
If this now takes an additional parameter, you need to expose that as a new overload and make all parameters here have no default values + mark it as hidden. See https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-mocking-factory-builder-methods

Reply:
Yes, I will do that. Although my plan was to do so as a separate PR to resolve https://github.com/Azure/azure-sdk-for-net/issues/31272. What is the team's preference for partitioning work?

Reply:
Up to you, but at least making sure there's tracking items. Personally, something like this I would've included here because you're already changing the model factory, so doing it in one shot makes it easier not to forget later. A change like this won't show up in APIView, so it'd be hard to draw the conclusion that another method should've been added.

Reply:
Got it. I will keep this in mind going forward. Thanks!